### PR TITLE
Fix nveto readout

### PR DESCRIPTION
**Pull request by petergaemers**
Missed old way to refer to writing out nv data.
nv is always in data type whether you want it or not. Use the targets option instead

### CODE REVIEW

**Comment by ramirezdiego** [comment]
Wait, was this supposed to trigger an error? I'm quite sure I tested #125 right before merging. Have you tested now a dummy TPC+nVeto run?

**Comment by petergaemers** [comment]
It crashes for just TPC

**Comment by petergaemers** [comment]
@ramirezdiego Oke this is the final one. Sorry for the confusion

File: wfsim/strax_interface.py [hunk 1]
```diff
@@ -714,13 +714,20 @@ def compute(self):
...
                     data_type=data_type)
-            else:
+                #If nv is not one of the targets just return an empty chunk
+            if 'nv' in data_type:
+                chunk[data_type] = self.chunk(
+                    start=self.sim.chunk_time_pre,
+                    end=self.sim.chunk_time,
+                    data=np.array([]),
+                    data_type=data_type)
+            elif 'nv' not in data_type:
```
Comment (ramirezdiego) [inline wfsim/strax_interface.py:730]: I don't understand the logic here. Why not...?
```
if 'nv' in data_type:
    if 'nveto' in self.config['targets']:
        bla
    #If nv is not one of the targets just return an empty chunk
    else:
        blabla
else:
    blablabla
```

The current code would redefine `chunk[data_type]` twice, if nveto is one of the targets, wouldn't it?

Comment (ramirezdiego) [reply wfsim/strax_interface.py]: Fixed here: https://github.com/XENONnT/WFSim/pull/147/commits/322807f2faaeea961d0c40e81454abc846794187

**Review by zhut19 — APPROVED** [review]
I tested running with target set to both `('tpc')` and `('tpc', 'nveto')`, both works

**Pull request merged**